common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -313,6 +352,43 @@ public boolean isActive() {
 
         @Override
         protected void renderBar(PoseStack poseStack, float renderY, float renderHeight, float progress) {
+            if (progress > 1) { // overflowing mana
```

Comment:
Bit of duplicate code tbh, do you think we can do anything with it? If not, this is fine I guess.

Reply:
Maybe it is better not to abstract this into hell

Reply:
Can't do that for now

Reply:
Okay, I agree